src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import React, {Component} from 'react';
+import sCompare from 'react-addons-shallow-compare';
+import {pick} from 'lodash';
+import SplitPane from 'react-split-pane';
+
+import {flux} from '../../Firefly.js';
+import {LO_VIEW, getLayouInfo} from '../../core/LayoutCntlr.js';
+import {TablesContainer} from '../../tables/ui/TablesContainer.jsx';
+import {ChartsContainer} from '../../charts/ui/ChartsContainer.jsx';
+import {VisToolbar} from '../../visualize/ui/VisToolbar.jsx';
+import {TriViewImageSection} from '../../visualize/ui/TriViewImageSection.jsx';
+import {createContentWrapper} from '../../ui/panel/DockLayoutPanel.jsx';
+
+export class LcResult extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    shouldComponentUpdate(np, ns) {
+        return sCompare(this, np, ns);
+    }
+
+    componentDidMount() {
+        this.removeListener = flux.addListener(() => this.storeUpdate());
+    }
+
+    componentWillUnmount() {
+        this.removeListener && this.removeListener();
+    }
+
+    storeUpdate() {
+        const nextState = pick(getLayouInfo(), ['title', 'mode', 'showTables', 'showImages', 'showXyPlots', 'showForm', 'images']);
+        this.setState(nextState);
+    }
+
+    render() {
+        const {title, mode, showTables, showImages, showXyPlots, showForm=true, searchDesc, images} = this.state;
+        var {expanded, standard, closeable} = mode || {};
+        const content = {};
+        var visToolbar;
+        if (showImages) {
+            visToolbar = <VisToolbar key='res-vis-tb'/>;
+            content.imagePlot = (<TriViewImageSection key='res-tri-img'
+                                        closeable={closeable}
+                                        imageExpandedMode={expanded===LO_VIEW.images}
+                                        {...images}  />);
+        }
```

Comment:
I don't think you want a TriViewImageSection here.  Must a MultiImageViewer.  If we want coverage and user added image in the LCViewer the we probably need to generalize TriViewImageSection to another component.